### ISSUE TITLE
Relative deps configuration as a separated file

### ISSUE THREAD

**Issue by dvh91** [mweststrate/relative-deps#30]
First off, it's a great and useful library! I'm using it and enjoying every minute.

I think it could be better to hold the relative deps config in a different file which will be possible to add to `.gitignore` rather than holding it in `package.json`.

When working with multiple packages with few devs working on the same project, sometimes not everyone has the same repos structure locally and it's possible for some people to only have a subset of the relative deps and not all of them (which now fails the process when the first relative dep is not found).

**Comment by mweststrate** [comment]
Makes sense! Feel free to create a PR that supports that pattern. Please do
make sure the old approach works as well :)
